.github/workflows/build-prod-app.yml

```diff
@@ -70,3 +70,15 @@ jobs:
         with:
           name: Build Artifacts
           path: app/build/outputs/
+      - name: Create service_account.json
+        run: echo '${{ secrets.SERVICE_ACCOUNT_JSON }}' > service_account.json
+      - name: Deploy to production track
+        uses: r0adkll/upload-google-play@v1
+        with:
+          serviceAccountJson: service_account.json
+          packageName: com.wire
+          releaseFiles: app/build/outputs/bundle/prodCompatRelease/*.aab
+          track: prod
```

Comment:
if we are trying to keep the same behavior as Jenkins then please do not deploy directly onto prod track, afaik Jenkins uses Alpha track